webpack.config.js

```diff
@@ -337,11 +339,9 @@ module.exports = async (env, argv) => {
           }
         },
         {
-          // We reference the sources of some libraries directly, and they use async/await,
-          // so we have to run it through babel in order to support the Samsung browser on Oculus Go.
-          test: [path.resolve(__dirname, "node_modules/naf-janus-adapter")],
```

Comment:
Does this comment and test for `naf-janus-adapter` no longer apply?

Reply:
Yup, support.js and its direct dependencies are the only files that need to be loaded with this legacy babel config now, and support.js does not require anything related to three.js, aframe or naf.